src/modules/search-result/search-result.md

```diff
@@ -0,0 +1,41 @@
+### Loading state
+
+```js
+<SearchResult id="example-01" aria-busy="true">
+  <p data-line-count="2"></p>
+</SearchResult>
+```
+
+
+### Regular state
+
+```js
+<SearchResult
+  id="example-02"
```

Comment:
when I use article 82984538 the get pdf is not fully stretched
![image](https://user-images.githubusercontent.com/12775440/96710534-7ee9a780-139c-11eb-8e64-c11f4af80e7f.PNG)

Reply:
That's terrible. I don't understand why this happens but I will have a look. Thank you.

Reply:
It's weird, it doesn't happen to me and seems to be some cache problem.

The border is not applied to the container but to the image and even if `box-sizing: border-box` is not applied (but should, I forgot about it), the caption below just stretches to the full width.

In your example, the border goes around what makes me think something goes wrong.

Reply:
Oh I am sorry I found out this issue while working on my [PR](https://github.com/oacore/design/pull/352). Unfortunately, I used the wrong target branch. It seems it was your previous branch and now you removed it or I didn't fetch new commits. I don't know. Sorry for confusing you. The issue is not present here.